courses/serializers.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+from rest_framework import serializers
+
+from .models import Course, Module
+
+
+class JsonListField(serializers.Field):
+    def to_representation(self, value):
+        if isinstance(value, unicode) or isinstance(value, str):
+            try:
+                value = json.loads(value)
+            except ValueError:
+                raise serializers.ValidationError("Must provide valid JSON.")
+
+        if value is not None and not isinstance(value, list):
+            raise serializers.ValidationError("This field must be a JSON array")
+
+        return value
+
+    def to_internal_value(self, obj):
+        return obj
+
+
+class CourseSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = Course
+        fields = ('uuid', 'title', 'author_name', 'overview', 'description', 'video_url', 'edx_instance',
+                  'price_per_seat_cents', 'url')
+
+
+class ModuleSerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
I'm fine either way, but we should talk as a group about whether HATEOAS is something we want in the API